create_ph5.sh

```diff
@@ -36,4 +36,13 @@ cd error
 mstoph5 -n master.ph5 -r ../../../samplerate/8H.10075..GH1.2012-08-27T23.01.00.ms
 keftoph5 -n master.ph5 -k ../../../samplerate/Das_SampleRate_error.kef
 
+cd ../..
+mkdir sampleratemultiplier0
+cd sampleratemultiplier0
+initialize_ph5 -n master.ph5
+keftoph5 -n master.ph5 -k ../../metadata/experiment.kef
+segdtoph5 -n master.ph5 -r ../../segd/1111.0.0.fcnt
+echo "y" | delete_table -n master.ph5 -D 1X1111
+keftoph5 -n master.ph5 -k ../../metadata/Das_t_1X1111.0.0_SRM0.kef
+
```

Comment:
ph5_no_srm can't be move to `/PH5/ph5/test_data/ph5` because `/PH5/ph5/test_data/ph5` will be deleted whenever TravisCI test is built. I actually start with having ph5_no_srm under that folder, and the result was I had to recreate ph5_no_srm after running create_ph5.sh

I will look at ph5tokef to make sure it run correctly.

Reply:
@timronan I think I should have ph5tokef to create kef file with the exact info from the table in the ph5 file, which means if the table have no 'sample_rate_multiplier', kef file will have no 'sample_rate_multiplier'. Do you agree with that approach?